portafly/README.md

```diff
@@ -0,0 +1,29 @@
+# PortaFly
+A React SPA crafted with the ambitous purpose of replacing Porta UI, currently residing inside the Rails App.
+
+## Available Scripts
+
+Inside the `portafly` directory, you can run:
+
+### `yarn start`
+
+Runs the app in the development mode.<br />
```

Comment:
Why are we doing this now? 😄

Reply:
Will remove it, it was acquired from create react app readme